app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,5 @@
+/*
+ *= require_tree .
+ *= require social-share-button
+ *= require_self
+ */
```

Comment:
Hey @tomhughes! 

I had to create a `app/assets/stylesheets/application.css` file for the `social-share-button` it was part of the requirements. 

https://github.com/huacnlee/social-share-button